mmv1/third_party/terraform/website/docs/d/composer_user_workloads_secret.html.markdown

```diff
@@ -0,0 +1,63 @@
+---
+subcategory: "Cloud Composer"
+description: |-
+  User workloads Secret used by Airflow tasks that run with Kubernetes Executor or KubernetesPodOperator.
+---
+
+# google\_composer\_user\_workloads\_secret
+
+Provides access to Kubernetes Secret configuration for a given project, region and Composer Environment.
+
+~> **Warning:** This data source is in beta, and should be used with the terraform-provider-google-beta provider.
+See [Provider Versions](https://terraform.io/docs/providers/google/guides/provider_versions.html) for more details on beta resources.
+
+## Example Usage
+
+```hcl
+resource "google_composer_environment" "example" {
+    name = "example-environment"
+    config{
+        software_config {
+            image_version = "composer-3-airflow-2"
+        }
+    }
+}
+
+resource "google_composer_user_workloads_secret" "example" {
+    environment = google_composer_environment.example.name
+    name = "example-secret"
+    data = {
+        username: base64encode("username"),
+        password: base64encode("password"),
+    }
+}
+
+data "google_composer_user_workloads_secret" "example" {
+    environment = google_composer_environment.example.name
+    name = resource.google_composer_user_workloads_secret.example.name
+}
+
+output "debug" {
+    value = data.google_composer_user_workloads_secret.example
+}
+```
+
+## Argument Reference
```

Comment:
this is missing documentation for the `data` field. you could copy the docs from the resource - but for data sources we often just refer to the resource docs instead of restating them. For example, see https://registry.terraform.io/providers/hashicorp/google/latest/docs/data-sources/bigquery_dataset#attributes-reference